### DISCUSSION TITLE
Can't access the file hosted no IPFS

### DISCUSSION THREAD

**Discussion by Aashim-Limbu** [Cyfrin/foundry-full-course-cu#2998] [Q&A]
i have files uploaded in IPFS node . i have the CID of it but i can't access the file from browser.
![image](https://github.com/user-attachments/assets/a58be907-3efc-4365-a8b2-e6b0f073c144)
Even though i try it in brave [Before pressing enter]
![image](https://github.com/user-attachments/assets/aab5ae95-b843-4f9a-abe1-f84c2a75d552)
After pressing Enter
![image](https://github.com/user-attachments/assets/85f9b9bf-5400-42a8-8aeb-c353c4dffb85)
Also i see  that IPFS companion is offline i guess
![image](https://github.com/user-attachments/assets/0691feb1-52e3-46f4-8d84-95e9ae703d16)
even though i have the IPFS desktop running
![Uploading image.png…]()

**Comment by EngrPips** [comment]
Hello @Aashim-Limbu, use this `https://ipfs.io/ipfs/${YOUR_CID}` and let's see if it will help.

**Comment by Aashim-Limbu** [comment]
@EngrPips yes it works but why ipfs:// isn't working? .Why do we need to use https:// instead of ipfs:// .since by using this we're making a request from server to ipfs node. isn't it ?

**Reply by EngrPips** [reply]
I am unsure why it didn't work for you with the `ipfs://${your CID} as I used that on my Chrome browser while my IPFS Desktop was running. Also, my IPFS Chrome extension is active, and it works fine. I heard Brave browser is IPFS compatible, so it should work, or maybe your browser needs to be updated.

**Reply by Aashim-Limbu** [reply]
@EngrPips  can you show the ipfs companion when it is online . 😂😂. I'm curious to see how will it look.

**Reply by EngrPips** [reply]
> @EngrPips can you show the ipfs companion when it is online . 😂😂. I'm curious to see how will it look.

![Screenshot 2024-12-06 at 12 00 09](https://github.com/user-attachments/assets/9577ba8d-3fbf-4fe9-a9b1-18a62da2c5d6)

**Reply by EngrPips** [reply]
That is how it looks for me.

**Reply by Aashim-Limbu** [reply]
Oh so ipfs:// protocol works for you isn't it .Otherwise i also have the same .
![image](https://github.com/user-attachments/assets/2eee4455-76ad-4d53-9ccd-92008a623a8e)

**Reply by EngrPips** [reply]
Yeah, it does work for me, I have IPFS Desktop installed on my Mac though.

**Comment by Aashim-Limbu** [comment]
@EngrPips also i would love to ask you a question. 
Let's say i have deployed a contract in sepolia and clear out the terminal without taking note . how do i search that contract in etherscan.how do i get the contract address .Or should i re-deploy it to get the address (That's the only way) .🤔

**Reply by EngrPips** [reply]
If you had deployed using Foundry, then you should be able to find the contract address in the `broadcast` folder, You don't have to deploy a new contract.

**Reply by Aashim-Limbu** [reply]
Oh nice thanks for the info though.😁👍

**Reply by EngrPips** [reply]
Anytime, Friend.
